app/controllers/stripe-setup/government-entity-document/get.controller.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+const { response } = require('../../../utils/response')
+const { isSwitchingCredentialsRoute, isAdditionalKycDataRoute, getCurrentCredential } = require('../../../utils/credentials')
+const { getAlreadySubmittedErrorPageData } = require('../stripe-setup.util')
+
+module.exports = (req, res, next) => {
```

Comment:
Should name the controller function rather than having it anonymous to adhere to our style guide